action.yml

```diff
@@ -6,8 +6,9 @@ inputs:
     required: false
     default: ${{ github.token }}
   notify-after:
-      description: 'The time after which unreleased commits should be considered stale and should notify for a release'
-      required: false
+    description: 'The time after which unreleased commits should be considered stale and should notify for a release'
+    required: false
+    default: 7
```

Comment:
I believe this default set here messed up some logic we have internally. To be extra sure, I would remove it